openpype/hosts/traypublisher/plugins/create/create_mov_batch.py

```diff
@@ -0,0 +1,191 @@
+import copy
+import os
+import re
+
+from openpype.client import get_assets, get_asset_by_name
+from openpype.hosts.traypublisher.api import pipeline
+from openpype.lib import FileDef, get_subset_name_with_asset_doc
+from openpype.pipeline import (
+    CreatedInstance,
+    CreatorError
+)
+
+from openpype.hosts.traypublisher.api.plugin import TrayPublishCreator
+
+
+class BatchMovCreator(TrayPublishCreator):
+    """Creates instances from .mov file(s)."""
+    identifier = "render_mov_batch"
+    label = "Batch Mov"
+    family = "render"
+    description = "Publish batch of movs"
+
+    create_allow_context_change = False
+    version_regex = re.compile(r"^(.+)_v([0-9]+)$")
+
+    def __init__(self, project_settings, *args, **kwargs):
+        super(BatchMovCreator, self).__init__(project_settings,
+                                              *args, **kwargs)
+        self.default_variants = (project_settings["traypublisher"]
+                                                 ["BatchMovCreator"]
+                                                 ["default_variants"])
+
+        self.default_tasks = (project_settings["traypublisher"]
+                                              ["BatchMovCreator"]
+                                              ["default_tasks"])
+
+        self.extensions = (project_settings["traypublisher"]
+                                           ["BatchMovCreator"]
+                                           ["extensions"])
```

Comment:
Maybe prepare variable with this plugin settings
```suggestion
        creator_settings = (
            project_settings["traypublisher"]["BatchMovCreator"]
        )
        self.default_variants = creator_settings["default_variants"]
        self.default_tasks = creator_settings["default_tasks"]
        self.extensions = creator_settings["extensions"]
```